### PR TITLE
New Series: AI Explainability Series

### PR DESCRIPTION
**Pull request by cobusbernard**
# Content Pull Request Checklist

Please check to make sure your content submission meets the following requirements by adding an `x` between the brackets:

- [x] The title of this pull request reflects the title of my content.
- [ ] I have reviewed the [Content Review Checklist](https://github.com/build-on-aws/content/blob/main/CONTENT_REVIEW_CHECKLIST.md).
- [x] This pull request contains one piece of content (post or page).

Is this pull request linked to a Content Proposal or Idea Suggestion Issue? If so, please link to it below using the number or full link.

```text
Resolves #5

or

Content for https://github.com/build-on-aws/content/issues/5
```

**By submitting this pull request, I confirm I own this content, have permission to use each image included in this content, and that it will be released under the [CC BY-SA 4.0 SA License](/LICENSE).**

### CODE REVIEW

**Comment by cobusbernard** [comment]
Review feedback from #368 :

Review Notes @mikegc (first pass only):

Where does the CNN image come from? I think it's here: https://www.mdpi.com/1424-2818/12/1/29 which references it as "Figure 1. Deep neural networks learn hierarchical feature representations. After (LeCun et al. (2015))" Use that?
LIne 33: "should be able to expledata and a diagnostics" - help me untie this typo pls.
Line 39: I added "Artificial Intelligence eXplainability" to AIX, but should we just stick to "explainability" or does this instance need AIX? AIX is not used anywhere else in the article.
Line 39: Missing link to ref? https://arxiv.org/pdf/2011.07876.pdf? - I HAVE PROPOSED FIX.
Line 61: For readability, what is this section? Why is this section?
Line 65: "X π={x 1 ,…,x k } to prediction \mathcal{Y}={y^1, \dots, y^n}Y={y 1,…,y n}" - Uses 'x k' and 'y n'. Should these be consistent?
Line 75: Open question: Is 'math' supported by the BuildOn platform? - Asked in Slack.
Line 174: Suggested links with references and a little more details as this is a 'References' section not a 'links' section. (I only looked at the first 3)

**Comment by cobusbernard** [comment]
New PR from #322 and #368, please continue review here. Due to making the repo public.

@mikegc-aws : You should be able to continue the review & updates here after we resolved your permissions, please let me know if you still have issues.

File: posts/ai-explainability/01-ai-explainability-intro/index.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+## What is explainability
+
+In its core, model explainability is either  to make decision process of a model understandable for humans or to provide an explanation for a single decision on a single data instance. In the case of black-box explanations, a simpler model can explain decisions made by a more complex model, we therefore can enjoy accuracy of more complex but opaque model, while explain its behaviour consistently with a simpler and less accurate model. There are practical examples for explainability, a trading system should be able to expledata and a diagnostics. Face recognition systems should explain why an image was matched to an identity. This should happen in a way humans can understand the reasoning. In a face recognition system, for instance we can use latent vector representations in the higher layers as "super pixels" where the intermediary images do make sense for human observer. in Figure 1, we can observe the convolutional layers gradually produce representations that are closer to what we can understand.
```
Comment (kirannsa) [inline posts/ai-explainability/01-ai-explainability-intro/index.md:32]: + a trading system should be able to expledata and a diagnostics. (Needs correction here i.e a trading systems should be able to explain ?)

**Comment by cobusbernard** [comment]
Closing the PR with the move to the post editor, content backed up.

**Pull request closed without merging**